<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qwad</class>
 <widget class="QMainWindow" name="Qwad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>296</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>481</width>
    <height>296</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>296</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Qwad - a WAD management tool</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Qwad.qrc">
    <normaloff>:/icons/wad.png</normaloff>:/icons/wad.png</iconset>
  </property>
  <property name="locale">
   <locale language="Spanish" country="Spain"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>481</width>
      <height>251</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="unpacktab">
     <attribute name="title">
      <string>Unpack WAD</string>
     </attribute>
     <widget class="QLineEdit" name="MuestraRutaWad">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaWad">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ruta hacia el archivo WAD que quieres desempaquetar</string>
      </property>
      <property name="text">
       <string>WAD file</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="MuestraRutaExtraer">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaExtraer">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Carpeta donde se almacenaran los contenidos del archivo WAD</string>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="Desempaqueta">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Unpack</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="TMDviewer">
     <attribute name="title">
      <string>TMD viewer</string>
     </attribute>
     <widget class="QLineEdit" name="TMDfilepath">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>10</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="TMDfilebutton">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ruta hacia el archivo WAD que quieres desempaquetar</string>
      </property>
      <property name="text">
       <string>TMD file</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>451</width>
        <height>151</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Title ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="TitleID">
         <property name="text">
          <string>0000000000000000</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>ID (ascii)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLineEdit" name="idASCII">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Version</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Version">
         <property name="text">
          <string>0</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Title type</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLineEdit" name="TitleType">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>IOS needed</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Group ID</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="4">
        <widget class="QLineEdit" name="GroupID">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Boot index</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="BootIndex">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="3">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Access rights</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="4">
        <widget class="QLineEdit" name="AccessRights">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Contents</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="Contents">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="IOSversion"/>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="packtab">
     <attribute name="title">
      <string>Pack WAD</string>
     </attribute>
     <widget class="QLineEdit" name="MuestraRutaEmpaquetado">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="MuestraRutaDesempaquetado">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>40</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaEmpaquetado">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Carpeta y nombre del nuevo WAD</string>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QToolButton" name="BotonRutaDesempaquetado">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>40</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Ruta hacia la carpeta que empaquetar en WAD</string>
      </property>
      <property name="text">
       <string>Folder</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Empaqueta">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Pack</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="NUStab">
     <attribute name="title">
      <string>Download from NUS</string>
     </attribute>
     <widget class="QLineEdit" name="NusOutputPath">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>241</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="Download_from_NUS">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>180</y>
        <width>121</width>
        <height>25</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Download</string>
      </property>
     </widget>
     <widget class="QToolButton" name="NusOutputButton">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>91</width>
        <height>25</height>
       </rect>
      </property>
      <property name="toolTip">
       <string>Carpeta y nombre del nuevo WAD</string>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Output</string>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
      <property name="arrowType">
       <enum>Qt::NoArrow</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="enteredTitleID">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>45</y>
        <width>161</width>
        <height>31</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>000000000000000000</string>
      </property>
     </widget>
     <widget class="QLabel" name="TitleIDlabel">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>51</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="locale">
       <locale language="English" country="UnitedStates"/>
      </property>
      <property name="text">
       <string>Title ID</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
      </property>
     </widget>
     <widget class="QLabel" name="Title">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Choose title:</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>125</x>
        <y>10</y>
        <width>291</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Custom title ID</string>
       </property>
      </item>
     </widget>
     <widget class="QCheckBox" name="decryptCheck">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>120</y>
        <width>171</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Decrypt</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="pack_in_WAD_checkbox">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>120</y>
        <width>181</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Pack into WAD</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="VersionlineEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>45</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>unset</string>
      </property>
     </widget>
     <widget class="QLabel" name="versionlabel">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Version</string>
      </property>
     </widget>
     <widget class="QLabel" name="availableLabel">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>79</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Available Versions</string>
      </property>
     </widget>
     <widget class="QLabel" name="availableVersions">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>121</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>unset</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAyuda">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAcerca_de_Qwad"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuAyuda"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAcerca_de_Qwad">
   <property name="icon">
    <iconset resource="../Qwad.qrc">
     <normaloff>:/icons/wad.png</normaloff>:/icons/wad.png</iconset>
   </property>
   <property name="text">
    <string>About Qwad</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="../Qwad.qrc">
     <normaloff>:/icons/qt4.png</normaloff>:/icons/qt4.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAbout_Wii_Signer">
   <property name="text">
    <string>About Wii Signer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../Qwad.qrc"/>
 </resources>
 <connections/>
</ui>
